dispatcher/frontend-ui/src/components/CloneSchedule.vue

```diff
@@ -32,7 +32,7 @@
         if (!this.ready)
           return;
         let parent = this;
-        let payload = {name: this.form_name};
+        let payload = {name: this.form_name.trim()};
```

Comment:
I think `ready()` should do the `trim()` as well so that a new name such as ` ` doesn't trigger the clone button.
Also, we should maybe update `this.form_name` so that in case of error, the user input is updated as well.